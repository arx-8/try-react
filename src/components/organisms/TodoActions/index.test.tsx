import { _TodoActions as SutComponent } from "."
import { create } from "react-test-renderer"
import React from "react"

describe("Snapshot", () => {
  it("all", () => {
    // ## Arrange ##
    // ## Act ##
    const result = create(
      <SutComponent
        setVisibilityFilter={() => {}}
        currentVisibilityFilter="all"
      />
    )
    // ## Assert ##
    expect(result).toMatchSnapshot()
  })
})
