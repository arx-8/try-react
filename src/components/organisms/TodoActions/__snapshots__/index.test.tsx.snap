// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot all 1`] = `
.emotion-0 {
  margin-top: 8px;
  margin-left: 8px;
  margin-right: 8px;
}

<div
  className="emotion-0"
>
  <fieldset
    className="MuiFormControl-root"
  >
    <legend
      className="MuiFormLabel-root"
    >
      Actions
    </legend>
    <div
      aria-label="position"
      className="MuiFormGroup-root MuiFormGroup-row"
      role="radiogroup"
    >
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-25 MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-checked-26 Mui-checked MuiIconButton-colorPrimary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <div
              className="PrivateRadioButtonIcon-root-41 PrivateRadioButtonIcon-checked-43"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root PrivateRadioButtonIcon-layer-42"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                />
              </svg>
            </div>
            <input
              checked={true}
              className="PrivateSwitchBase-input-28"
              disabled={false}
              name="position"
              onChange={[Function]}
              type="radio"
              value="all"
            />
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          All
        </span>
      </label>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-25 MuiRadio-root MuiRadio-colorPrimary MuiIconButton-colorPrimary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <div
              className="PrivateRadioButtonIcon-root-41"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root PrivateRadioButtonIcon-layer-42"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                />
              </svg>
            </div>
            <input
              checked={false}
              className="PrivateSwitchBase-input-28"
              disabled={false}
              name="position"
              onChange={[Function]}
              type="radio"
              value="active"
            />
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Active
        </span>
      </label>
      <label
        className="MuiFormControlLabel-root"
      >
        <span
          aria-disabled={false}
          className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-25 MuiRadio-root MuiRadio-colorPrimary MuiIconButton-colorPrimary"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={null}
        >
          <span
            className="MuiIconButton-label"
          >
            <div
              className="PrivateRadioButtonIcon-root-41"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
              </svg>
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root PrivateRadioButtonIcon-layer-42"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                />
              </svg>
            </div>
            <input
              checked={false}
              className="PrivateSwitchBase-input-28"
              disabled={false}
              name="position"
              onChange={[Function]}
              type="radio"
              value="completed"
            />
          </span>
        </span>
        <span
          className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
        >
          Completed
        </span>
      </label>
    </div>
  </fieldset>
</div>
`;
